Examples/Standups/Standups/Dependencies/SoundEffectClient.swift

```diff
@@ -0,0 +1,45 @@
+import AVFoundation
+import Dependencies
+
+struct SoundEffectClient {
```

Comment:
realized we never implemented sound effects from scrumdinger so took a stab at that.